### DISCUSSION TITLE
Flash ESP32 CAM bloked

### DISCUSSION THREAD

**Discussion by GBaud** [arendst/Tasmota#16651] [Q&A] [answered]
Hello to all,
since several days, I try in vain to flash an ESP32 Cam AI Thinker with tasmota in all possible ways: 
- With the micro-usb module
- With serial adapter
- With tasmotizer, online or with other flashers
- From a PC, a mac or a raspberry Pi
- With an older .bin than the current one. 
- other ESP32 cam, other cable...

But each time, I have the same result : the logs show that there is something running in a loop, and it never happens with a wifi network emission. 
I put you the log in attachment. 

If anyone sees what I'm missing... Thanks in advance!

[esp-web-tools-logs.txt](https://github.com/arendst/Tasmota/files/9640811/esp-web-tools-logs.txt)

**Comment by sfromis** [comment, marked as answer]
**Brownout detector was triggered** means
![image](https://user-images.githubusercontent.com/47082390/192146145-6519af5e-1279-4548-b6a7-ea3137451d8e.png)

**Reply by GBaud** [reply]
Thank you for the answer. Again, I tried several things: 
- leave the ESP 32 Cam on the serial adapter by removing the clable loop for the Boot
- a usb cable with two dupont connectors, on a 5V phone adapter (5W and 10W). 

In all these cases, the module is supposed to be powered in 5V and should work properly, right?

**Reply by sfromis** [reply]
More power means more power. No way around that. 5V is right, but should be at least 500 mA. If you have an ESP32-CAM with ESP32-CAM-MB, that adapter tends to make the already crappy board even worse.

**Reply by GBaud** [reply]
Very well, thank you for these precisions. I'll try to find out what's wrong to feed it better in this case!
